app/models/comment.rb

```diff
@@ -0,0 +1,2 @@
+class Comment < ApplicationRecord
+end
```

Comment:
- [ ] It will be great to have a method that updates the comments counter for a post. This is a major requirement for this project to be approved Hint: You can use active record call backs. 
Example `after_save` to call the `:update_comments_counter`method every time a new comment has been saved to database, then make the `:update_comments_counter` method private